### ISSUE TITLE
Does SAM 3.1 support SAM 1’s interactive image segmentation task (point/box-based)?

### ISSUE THREAD

**Issue by pank98-hash** [facebookresearch/sam3#555]
I have tested the official SAM 3.1 video example with only **geometric prompts (points/boxes)**. However, the model returns **"no object found"** and fails to generate any valid mask.
I would like to ask:
1. Has anyone else encountered the same issue when using only point/box prompts in SAM 3.1 video tasks?
2. Is the interactive segmentation capability degraded in SAM 3.1 compared to SAM 1/2 for pure geometric prompts?
3. Or does SAM 3.1 adopt a different mask post-processing pipeline (e.g., higher confidence threshold, stricter filtering, or changed prompt fusion logic) that makes it less sensitive to standalone point/box inputs?
Looking forward to your clarification.

test code：
```
# 1. load video
video_path = f"{sam3_root}/assets/videos/0001"
...
# 2. load model
# 3. process
response = predictor.handle_request(
    request=dict(
        type="start_session",
        resource_path=video_path,
    )
)

session_id = response["session_id"]
_ = predictor.handle_request(
    request=dict(
        type="reset_session",
        session_id=session_id,
    )
)

frame_idx = 0
obj_id = 2
points_abs = np.array(
    [
        [760, 550],  # positive click
    ]
)

labels = np.array([1])

points_tensor = torch.tensor(
    abs_to_rel_coords(points_abs, IMG_WIDTH, IMG_HEIGHT, coord_type="point"),
    dtype=torch.float32,
)
points_labels_tensor = torch.tensor(labels, dtype=torch.int32)

response = predictor.handle_request(
    request=dict(
        type="add_prompt",
        session_id=session_id,
        frame_index=frame_idx,
        points=points_tensor,
        point_labels=points_labels_tensor,
        obj_id=obj_id,
    )
)
out = response["outputs"]

plt.close("all")
visualize_formatted_frame_output(
    frame_idx,
    video_frames_for_vis,
    outputs_list=[prepare_masks_for_visualization({frame_idx: out})],
    titles=["SAM 3.1 Dense Tracking outputs"],
    figsize=(6, 4),
    points_list=[points_abs],
    points_labels_list=[labels],
)
```
